stores-performance/app.js

```diff
@@ -0,0 +1,333 @@
+});
+
+var timeDay = ['6am', '7am', '8am', '9am', '10am', '11 am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm', '6pm', '7pm'];
+var mainTable = document.getElementById('main-table'); //Get main table
+//Get element access <table></table>
+
+// var eRowItem = document.createElement('td');
+
+// }else if{
+// eRowItem.textContent = avgCookiesArray;
+// this.CookieArray.push(avgCookiesArray);
+// loopTr.appendChild(eRowItem);
+
+function loopRenderTable(){
+  for(var w = 0; w < stores.length; w++){
+    stores[w].renderTable();
+  }
+}
+
+//Cookie store repo
+function CookieStore(name, minCust, maxCust, avgSale) {
+  this.name = name;
+  this.minCust = minCust; //maximum customers per hour
+  this.maxCust = maxCust; //minimum customers per hour
+  this.avgSale = avgSale; //avg sales per customery
+  this.cookieArray = [];
+  this.totalSales = 0;
+}
+
+//Our Stores
+var onePike = new CookieStore('First and Pike', 23, 65, 6.3); //First and Pike Store
+var seaTac = new CookieStore('Seatac Airport', 3, 24, 1.2); //Seatac Store
+var seaCenter = new CookieStore('Seattle Center', 11, 38, 3.7); //Seattle Center Store
+var capHill = new CookieStore('Capitol Hill', 20, 38, 2.3); //Capitol Hill Store
+var alki = new CookieStore('Alki', 2, 16, 4.6); //Alki Store
+
+var stores = [onePike, seaTac, seaCenter, capHill, alki];
+
+//RNGEESUS
+CookieStore.prototype.avgHourly = function(){
+  return Math.floor(Math.floor((Math.random() * (this.maxCust - this.minCust + 1) + this.minCust)) * this.avgSale);
+}; //Function for calculating random number based on each location's info.
+
+//CREATE PROTOTYPE THAT CREATES A TH FOR STORE TITLE AND TDS
+CookieStore.prototype.renderTable = function(){
+  var loopTr = document.createElement('tr');
+  var storeTh = document.createElement('th');
+  mainTable.appendChild(loopTr);
+  storeTh.textContent = this.name;
+  loopTr.appendChild(storeTh);
+  for(var h = 0; h < timeDay.length; h++){
+    var eRowItem = document.createElement('td');
+    var avgCookiesArray = this.avgHourly();
+    eRowItem.textContent = avgCookiesArray;
+    this.cookieArray.push(avgCookiesArray);
+    loopTr.appendChild(eRowItem);
+    this.totalSales = 0;
+  }
+  //measures all sales and saves them to their respective cookie store
+  for(var d = 0; d < this.cookieArray.length; d++){
+    this.totalSales += this.cookieArray[d];
+  }
+  //makes a final column for each store, which contains the totals of each row.
+  var totalColumn = document.createElement('td');
+  totalColumn.textContent = this.totalSales;
+  loopTr.appendChild(totalColumn);
+};
+
+var tableHead = document.createElement('thead');//<thead> </thead>
+mainTable.appendChild(tableHead); //<table> <thead></thead> </table>
+
+// spacer block for thead
+var spacerTh = document.createElement('th');//<thead> </thead>
+spacerTh.textContent = 'Stores';
+tableHead.appendChild(spacerTh); // //<thead> <th></th> </thead>
+
+//creates header with time range
+for(var h = 0; h < timeDay.length; h++){
```

Comment:
82-90 should be a standalone function to match the pattern you use for the rest of your functionality